recipes/virsorter/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set virsorterVersion = "1.0.5" %}
+{% set virsorterSha256 = "5a25f74cac02502a6a1961c31945ae1d6bae27e718f0a471a674bf549c376330" %}
+
+package:
+  name: virsorter
+  version: {{ virsorterVersion }}
+
+source:
+  url: https://github.com/simroux/VirSorter/archive/v{{ virsorterVersion }}.tar.gz
+  sha256: {{ virsorterSha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - perl-bioperl
```

Comment:
you probably want to have perl here as well

Reply:
done